tests/python/gaia-ui-tests/gaiatest/apps/system/regions/status_bar.py

```diff
@@ -16,11 +16,14 @@ class StatusBar(PageRegion):
 
     @property
     def height(self):
-        return self.root_element.rect['height']
+        if self.is_displayed:
+            return self.root_element.rect['height']
+        else:
+            return 0
 
     @property
     def is_displayed(self):
-        return self.root_element.is_displayed()
+            return self.root_element.is_displayed() and self.root_element.rect['y'] == 0
```

Comment:
yeah.. apparently .is_displayed() still returns true.  Hence in that case, rect['y'] will return a negative value.  I wonder whether someone can fix is_displayed() for us.